tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -975,6 +1009,7 @@ def on_environment_change(self, new_environment):
 
     def do_test(self, test):
         url = self.test_url(test)
+        self.protocol.test_path = test.path
```

Comment:
i think there is a better way to get that path than the way i'm doing it here. open to suggestions